core/terraform/airlock/data.tf

```diff
@@ -0,0 +1,21 @@
+data "local_file" "airlock_processor_version" {
+  filename = "${path.root}/../../airlock_processor/_version.py"
+}
+
+data "azurerm_private_dns_zone" "eventgrid" {
```

Comment:
Are dependencies ok here? I try not to use data in modules where the resource (i.e. DNS Zone) is deployed in the same operation and pass a var, to avoid terraform trying to read it before it exists.

Reply:
I just copied it from another location to this file. It worked fine for 3 years.

Reply:
Guess leave as is then :)